core/block/chats/service.go

```diff
@@ -260,14 +268,38 @@ func (s *service) AddMessage(ctx context.Context, sessionCtx session.Context, ch
 		return err
 	})
 	if err == nil {
-		go s.sendPushNotification(spaceId, chatObjectId, messageId, message.Message.Text)
+		spaceName := s.store.GetSpaceName(spaceId)
```

Comment:
I think it's better to move all code related to notification to `sendPushNotification` function